### ISSUE TITLE
Bug: Telegram plane without background

### ISSUE THREAD

**Issue by hannah23280** [FortAwesome/Font-Awesome#19325]
### Bug description

In v5,
   Telegram-plane is able to render just the plane , without any background. 
    https://fontawesome.com/v5/icons/telegram-plane?s=solid&f=brands

In v6,
   Such is not the case.  Whether i use telegram-plane or telegram, there is no way to achieve the above. 
 
Suspect this could be missing in v6.

### Font Awesome version

v6.2

### Serving

Self-hosted

### Implementation

CSS

### Browser and Operating System

-Edge v105
-Chrome v104
Windows 10 pro

### Web bug report checklist

- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

We try our best to meet the guidelines of third party brands

I do not exactly recall the history of the Telegram icon. I've tried to retrieve it from GitHub issues but it is time consuming on my side 

My best guess is that `telegram-plane` was an unofficial logo, and when Telegram released the official brand guidelines, we added the official logo. Then in FA6 we removed `telegram-plane` and aliased it to `telegram` to prevent breaking changes in FA5

Refs: #15614 #16664

I'm asking @sensibleworld if he recalls better than me

*...Said so...*

1. We accept request about unofficial logos for brands, so feel free to request a new icon
2. It is quite easy with the SVG+JS implementation to add icons from old versions of FA: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome#svg-framework

**Comment by hannah23280** [comment]
> Hi!
> 
> Thanks for being part of the Font Awesome Community.
> 
> We try our best to meet the guidelines of third party brands
> 
> I do not exactly recall the history of the Telegram icon. I've tried to retrieve it from GitHub issues but it is time consuming on my side
> 
> My best guess is that `telegram-plane` was an unofficial logo, and when Telegram released the official brand guidelines, we added the official logo. Then in FA6 we removed `telegram-plane` and aliased it to `telegram` to prevent breaking changes in FA5
> 
> Refs: #15614 #16664
> 
> I'm asking @sensibleworld if he recalls better than me
> 
> _...Said so..._
> 
> 1. We accept request about unofficial logos for brands, so feel free to request a new icon
> 2. It is quite easy with the SVG+JS implementation to add icons from old versions of FA: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome#svg-framework

Noted.  However the  'telegram' does not look the same as 'telegram-plane'. The former has a background, whereas the latter does not have.

Facebook  has **fa-facebook-f** to display the "F" without any background, and **fa-facebook-square** has a square background.   So i thought telegram should also have equivalent 2 displays - one that show a plane without background, and another with plane with background.

Hence I thought the "plane without background" could have been accidentally missed out in v6.

**Comment by LucaRed** [comment]
I agree with @hannah23280, the Telegram icon should come in two different versions, the official one and the plane-only version (just like `facebook` and `facebook-f`).

Here's the new `telegram-plane` icon, based on their current icon (which you can [download from here](https://telegram.org/press#telegram-logos)).

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><path d="M55.523 237.368c116.073-50.571 193.473-83.91 232.2-100.019 110.574-45.991 133.55-53.98 148.526-54.244 3.294-.058 10.658.758 15.429 4.629 4.028 3.269 5.136 7.684 5.666 10.783.53 3.099 1.191 10.158.666 15.674-5.992 62.96-31.92 215.744-45.11 286.259-5.581 29.838-16.571 39.842-27.21 40.821-23.123 2.128-40.68-15.28-63.075-29.96-35.044-22.972-54.841-37.272-88.857-59.688-39.311-25.905-13.827-40.143 8.576-63.412 5.863-6.09 107.74-98.755 109.711-107.16.247-1.052.476-4.97-1.852-7.04-2.329-2.07-5.765-1.362-8.244-.799-3.515.798-59.497 37.8-167.948 111.007-15.89 10.912-30.283 16.228-43.179 15.95-14.216-.308-41.563-8.039-61.893-14.647-24.935-8.105-44.752-12.39-43.027-26.156.9-7.17 10.773-14.503 29.621-21.998z"/></svg>
```
